.github/workflows/helm-release.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
           # Enable pipefail so git command failures do not result in null versions downstream
           set -x
 
-          echo "RELEASE_VERSION=$(date +'%Y.%-m.%-d-%H%M%S')" >> $GITHUB_OUTPUT
+          echo "RELEASE_VERSION=$(date +'%Y.%-m.%-d%H%M%S')" >> $GITHUB_OUTPUT
```

Comment:
the `%H%M%S` will **always** be 6 digits, so we can back into the "day" piece pretty consistently (6 chars from the right)